cities/queries/fips_query.py

```diff
@@ -631,16 +646,101 @@ def __init__(
         outcome_comparison_period=None,
         outcome_percentile_range=None,
     ):
-        super().__init__(
```

Comment:
So you seem to be repeating most of the __init__, except perhaps for

self.data = MSADataGrabber()
self.gdpVAR = "gdp_ma"

(anything else?)

Did you try calling super init and overwriting instead? Or some other way to avoid repeating so much code?